### DISCUSSION TITLE
Unable to apply for GitHub Education

### DISCUSSION THREAD

**Discussion by kuroneko6423** [community/community#122293] [GitHub Education] [answered]
### Hi everyone,

I am a student living in Japan and I am unable to submit my student ID card with the following error.
Also, there was nothing to correct regarding the billing information that is in error.
What should I do in this case?

![image.png](https://github.zendesk.com/attachments/token/uCu66uMD788CKw6B92HzHPnTV/?name=image.png)

**Comment by IsmaelMousa** [comment]
+1, Please notify me if anybody has a solution for this 🥲

**Comment by docjay1** [comment]
I submitted my student ID pictures front and back as expiration is on back side but still nothing. one page says its processing and other page states its rejected. Any help would be appreciated.

**Reply by abdullah-dev5** [reply]
Apply again it takes sometime when i applied it gets 2 days

**Reply by docjay1** [reply]
thanks will try again.

**Comment by Avotexs** [comment]
i think you need to go to your GitHub billing settings "https://docs.github.com/en/billing" and check if your full name matches with your academic affiliation document. and btw you can contact the support  (https://support.github.com/request)

**Reply by kuroneko6423** [reply]
fmm?

![image](https://github.com/community/community/assets/42853476/1836bd23-6b35-444c-a6bf-8156d51a6643)

**Comment by tstran155** [comment]
Hi kuroneko6423,

Your name in the academic affiliation document must match exactly with the name as it appears in your GitHub billing information. I suggest you visit the GitHub billing information page at 'https://github.com/settings/billing/payment_information' to update this. If the issue persists you might want to contact GitHub support. 

Good luck!

**Reply by docjay1** [reply]
it wont let me submit a ticket. Name is same as what my institution have on my ID card.

**Reply by tstran155** [reply, marked as answer]
I am sorry to hear that. You may try to submit the ticket from a different device, browser or network. The last resort is to take a screenshot and reach out to GitHub support directly via email (support@github.com).
